lib/js/components/Modals/Modal/Modal.stories.ts

```diff
@@ -24,7 +24,7 @@ const StoryTemplate: StoryFn<typeof Modal> = (args) => ({
 		':footerTertiaryButtonText="footerTertiaryButtonText" :footerTertiaryButtonIcon="ICONS[footerTertiaryButtonIcon]" :footerTertiaryButtonState="footerTertiaryButtonState" ' +
 		':footerCheckboxText="footerCheckboxText" :headerImage="headerImage">' +
 		'<div v-html="defaultSlot" />' +
-		'<template #rightActions><div style="display: inline-flex;" v-html="rightActionsSlot" /></template>' +
+		'<template #rightActions><div style="display: inline-flex;" v-if="rightActionsSlot" v-html="rightActionsSlot" /></template>' +
```

Comment:
if we want to disable this slot when `rightActionsSlot` is empty, then it's better to move condition to the `<template>`.
It doesn't matter here becasue there is no additional HTML markup for the slot, but in Vue empty slot is different than no slot